lib/src/base-styles/_nx-colors.scss

```diff
@@ -50,11 +50,36 @@
   --nx-color-validation-invalid-icon: var(--nx-color-validation-invalid);
   --nx-color-validation-invalid-message: var(--nx-color-validation-invalid);
 
-  --nx-color-alert-info: var(--nx-swatch-blue-40);
   --nx-color-alert-success: var(--nx-swatch-green-25);
```

Comment:
- --nx-color-alert-info: var(--nx-swatch-blue-40);
- --nx-color-alert-warning: var(--nx-swatch-orange-50);

I think above variables can't be removed as Ross's comments in another PR

> All CSS variables are part of our public API. They should be marked as deprecated in the docs (and in code comments here) and their values should be updated to be aliases for the equivalent new variables. The deprecation notice in the docs should indicate what variables should be used instead of these.

Reply:
> * --nx-color-alert-info: var(--nx-swatch-blue-40);
> * --nx-color-alert-warning: var(--nx-swatch-orange-50);
> 
> I think above variables can't be removed as Ross's comments in another PR
> 
> > All CSS variables are part of our public API. They should be marked as deprecated in the docs (and in code comments here) and their values should be updated to be aliases for the equivalent new variables. The deprecation notice in the docs should indicate what variables should be used instead of these.

Thanks, marked as deprecated here: [6be517f](https://github.com/sonatype/sonatype-react-shared-components/pull/1054/commits/6be517f00ab24c7e247df847068422bf6117b0fa)